lib/utils/request.js

```diff
@@ -12,6 +12,10 @@ function request (options) {
       method: options.method,
       url: options.url,
       timeout: options.timeout,
+			// Turn off the use of Buffer in nets
+			// in order to make this module compatible
+			// for Webpack builds.
+			// see: https://github.com/maxogden/nets
```

Comment:
CI failed because of linting, looks like you used tabs here instead of spaces?

I can fix it up for you no problem, just let me know if you’re busy